data/gui/widget/tab_container_default.cfg

```diff
@@ -0,0 +1,252 @@
+#textdomain wesnoth-lib
+###
+### Definition of a tabbed container
+### A widget that has multiple pages and a list of available pages (tabs)
+### When an item is selected from the list, the corresponding page is shown
+###
+
+[tab_container_definition]
+	id = "default"
+	description = "tab container with vertical tabs, default."
+
+	[resolution]
+
+		min_width = 0
+		min_height = 0
+
+		default_width = 0
+		default_height = 0
+
+		max_width = 0
+		max_height = 0
+		
```

Comment:
Several lines have trailing whitespace - `git diff` highlights them, but the other tools don't (and the whitespace check currently skips the GUI folder).

Reply:
That doesn't even count as trailing whitespace though. I really don't know what you people have against keeping the indentation uniform.

Reply:
> Several lines have trailing whitespace - git diff highlights them, but the other tools don't (and the whitespace check currently skips the GUI folder).

Removed.